src/app/containers/NewSignupContainer/useSignup.js

```diff
@@ -0,0 +1,159 @@
+import { useState } from 'react';
+import { srpVerify, srpAuth } from 'proton-shared/lib/srp';
+import { useApi, usePlans, useConfig, useApiResult, useModals } from 'react-components';
+import { queryCreateUser, queryDirectSignupStatus } from 'proton-shared/lib/api/user';
+import { auth, setCookies } from 'proton-shared/lib/api/auth';
+import { subscribe, setPaymentMethod, verifyPayment } from 'proton-shared/lib/api/payments';
+import { mergeHeaders } from 'proton-shared/lib/fetch/helpers';
+import { getAuthHeaders } from 'proton-shared/lib/api';
+import { getPlan, PLAN } from './plans';
```

Comment:
We should add the eslint rule to always put local imports at the bottom if it's not there